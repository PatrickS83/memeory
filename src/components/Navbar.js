import React from 'react';

const Navbar = () => (
  <nav className="navbar is-primary">
    <div className="navbar-brand">
      <a className="navbar-item" href="http://patrickschuelke.me/memeory/">
        <img
          style={{ maxHeight: '2.5rem' }}
          src="./img/memeory-logo-white.png"
          alt="Website Logo"
        />
      </a>
    </div>

    <div className="navbar-menu">
      <div className="navbar-end">
        <div className="navbar-item">
          <div className="field is-grouped">
            <p className="control">
              <a className="button is-primary" href="https://github.com/PatrickS83/memeory/">
                <span>Github</span>
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </nav>
);

export default Navbar;
