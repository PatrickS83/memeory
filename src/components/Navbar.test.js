import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Navbar';

describe('<Navbar/>', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper.length).toEqual(1);
  });

  it('renders the Navbar as saved in snapshot', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
