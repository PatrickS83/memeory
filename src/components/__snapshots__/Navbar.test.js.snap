// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar/> renders the Navbar as saved in snapshot 1`] = `
<nav
  className="navbar is-primary"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="https://bulma.io"
    >
      <img
        alt="Bulma: a modern CSS framework based on Flexbox"
        height="28"
        src="https://bulma.io/images/bulma-logo-white.png"
        width="112"
      />
    </a>
  </div>
  <div
    className="navbar-menu"
  >
    <div
      className="navbar-start"
    >
      <a
        className="navbar-item"
        href="#"
      >
        Home
      </a>
    </div>
    <div
      className="navbar-end"
    >
      <div
        className="navbar-item"
      >
        <div
          className="field is-grouped"
        >
          <p
            className="control"
          >
            <a
              className="button is-primary"
              href="#"
            >
              <span>
                Download
              </span>
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</nav>
`;
